lib/video_info/providers/aparat.rb

```diff
@@ -0,0 +1,137 @@
+class VideoInfo
+  module Providers
+    class Aparat < Provider
+      def self.usable?(url)
+        url =~ /((.*)aparat\.com\/v\/(.*))/
+      end
+
+      def provider
+        'Aparat'
+      end
+
+      def api_key
+        ""
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.